collector/collector/config_local.py

```diff
@@ -43,6 +54,17 @@ class Testing(Config):
     SQLALCHEMY_DATABASE_URI = 'postgres://postgres:postgres@localhost/testdb'
 
     SQLALCHEMY_TRACK_MODIFICATIONS = True
+    MAX_DAYS_KEEP_UNFILTERED_RECORDS = 5
```

Comment:
I would prefer this to either be kept the default (35) or remove it to use the main value inherited from config. The change should be done in the test (see comment on purging.py).

Reply:
ok